### PR TITLE
added missing functions to show prisoner photos when available in prisoner search

### PR DESCRIPTION
**Pull request by jonesj2193**
Fixed issue where no prisoner photos show in results of prisoner search.

### CODE REVIEW

File: server/routes/index.ts [hunk 1]
```diff
@@ -28,6 +29,24 @@ export default function routes(services: Services): Router {
...
+
+      const prisonApi = new PrisonApi(user.token)
+      const photoData = await prisonApi.getPhoto(prisonerNumber)
+
+      const oneDay = 86400 as const
+      res.setHeader('Cache-Control', `private, max-age=${oneDay}`)
+      res.setHeader('Content-Type', 'image/jpeg')
+
+      if (!photoData) {
+        res.sendFile('prisoner.jpeg', { root: `${services.applicationInfo.packageJsonPath}/assets/images` })
```
Comment (xoen) [inline server/routes/index.ts:44]: Is this file missing?

Comment (jonesj2193) [reply server/routes/index.ts:44]: No you can see within the repo there is a `assets/images` folder from the top level

Comment (jonesj2193) [reply server/routes/index.ts:44]: Oh yeah the placeholder prisoner photo is missing. I will bring that in now

**Review by Mjwillis — APPROVED** [review]
LGTM

**Comment by ushkarev** [comment]
the fallback prisoner image isn't being loaded from the correct location… i _think_ i've fixed it in #190 and copied over the tests from non-associations where this prisoner search functionality came from

**Pull request merged**